components/esp_modem/examples/modem_tcp_client/main/espat_module/sock_module.hpp

```diff
@@ -0,0 +1,24 @@
+/*
+ * SPDX-FileCopyrightText: 2025 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "esp_modem_config.h"
+#include "cxx_include/esp_modem_api.hpp"
+
+#pragma  once
+
+namespace sock_dce {
+
+class Module: public esp_modem::GenericModule {
```

Comment:
This being tightly coupled to esp-at could be renamed to reflect that.

Reply:
I use the same names for other implementations, so they can be easily replaced.